source/mem/README.md

```diff
@@ -0,0 +1,77 @@
+# mem
+
+In memory (`mem`) driver useful when you want to always includes your database schema migration along with your binary.
+
+
+## Usage
```

Comment:
Add a section that compares this to the `stub` source driver. The main differences I see:
* Uses locks
* Supports multiple source sets that are accessible via URI
* Different migration struct